modules/enet/doc_classes/NetworkedMultiplayerENet.xml

```diff
@@ -16,23 +16,27 @@
 			</description>
 		</method>
 		<method name="create_client">
 			<return type="int" enum="Error">
 			</return>
-			<argument index="0" name="ip" type="String">
+			<argument index="0" name="address" type="String">
 			</argument>
 			<argument index="1" name="port" type="int">
 			</argument>
 			<argument index="2" name="in_bandwidth" type="int" default="0">
 			</argument>
 			<argument index="3" name="out_bandwidth" type="int" default="0">
 			</argument>
+			<argument index="4" name="client_port" type="int" default="0">
+			</argument>
 			<description>
-				Create client that connects to a server at address [code]ip[/code] using specified [code]port[/code]. The given IP needs to be in IPv4 or IPv6 address format, for example: [code]192.168.1.1[/code]. The [code]port[/code] is the port the server is listening on. The [code]in_bandwidth[/code] and [code]out_bandwidth[/code] parameters can be used to limit the incoming and outgoing bandwidth to the given number of bytes per second. The default of 0 means unlimited bandwidth. Note that ENet will strategically drop packets on specific sides of a connection between peers to ensure the peer's bandwidth is not overwhelmed. The bandwidth parameters also determine the window size of a connection which limits the amount of reliable packets that may be in transit at any given time. Returns [code]OK[/code] if a client was created, [code]ERR_ALREADY_IN_USE[/code] if this NetworkedMultiplayerEnet instance already has an open connection (in which case you need to call [method close_connection] first) or [code]ERR_CANT_CREATE[/code] if the client could not be created.
+				Create client that connects to a server at address [code]ip[/code] using specified [code]port[/code]. The given IP needs to be in IPv4 or IPv6 address format, for example: [code]192.168.1.1[/code]. The [code]port[/code] is the port the server is listening on. The [code]in_bandwidth[/code] and [code]out_bandwidth[/code] parameters can be used to limit the incoming and outgoing bandwidth to the given number of bytes per second. The default of 0 means unlimited bandwidth. Note that ENet will strategically drop packets on specific sides of a connection between peers to ensure the peer's bandwidth is not overwhelmed. The bandwidth parameters also determine the window size of a connection which limits the amount of reliable packets that may be in transit at any given time. Returns [code]OK[/code] if a client was created, [code]ERR_ALREADY_IN_USE[/code] if this NetworkedMultiplayerEnet instance already has an open connection (in which case you need to call [method close_connection] first) or [code]ERR_CANT_CREATE[/code] if the client could not be created. If [code]client_port[/code] is specified, the client will also listen to the given port, this is useful in some NAT traveral technique.
```

Comment:
Create client that connects to a server at *[code]address[/code]* using specified [code]port[/code]. The given *address* needs to be *either a fully qualified domain name (like [code]www.example.com[/code] or an IP address* in IPv4 or IPv6 format, for example: [code]192.168.1.1[/code].